frontend/packages/schema-model/src/lib/build-json-schema.ts

```diff
@@ -120,5 +120,21 @@ export const buildJsonSchema = (nodes: UiSchemaNodes): JsonSchema => {
         JSONPointer.set(out, jsonPointer, undefined);
       }
     });
-  return out;
+
+  return validateObject(out);
 };
+
+const validateObject = (object: JsonSchema): JsonSchema => {
+  if (!object.properties) { return object; }
+  const keys = Object.keys(object.properties);
+  const firstKey = keys[0];
+  if (object.properties && object.properties[firstKey] && object.properties[firstKey].type === "array") {
+    const items = object.properties[firstKey].items;
+    const properties = object.properties[firstKey].properties;
+    if (items && properties) {
+      items.properties = properties;
+      delete object.properties[firstKey].properties;
+    }
+  }
+  return object;
+};
```

Comment:
While this may solve the "symptom", it does not solve the underlying problem which is that the UISchema is not correct when creating an object with fields in the data model and then converting it to an array.

The issue seems to be that the pointers for the array node's children are not updated with the correct path. The actual code that does the conversion from UISchemaNode to JsonSchema works as expected, and is covered by many many unit tests. 
If you take a look at the `toggleArrayField` method of `ui-schema-reducers.ts` file, you will see that it only toggles the `isArray` flag. This is where we need to make changes.

What we want it to do is to update the "children" pointers (both in the `children` array and for each of the actual child nodes) to either:
- include `items` in the pointer path if _isArray is toggled on_, or
- remove `items` from the pointer path if _isArray is toggled off_.

F.ex, consider the following schema nodes, which represent an object with 2 fields (there are a lot more properties that are not so relevant for this issue that I have not shown):
```json
[
    {
        "pointer": "#/properties/group",
        "children": [
            "#/properties/group/properties/f1",
            "#/properties/group/properties/f2"
        ],
        "type": "object",
        "isArray": false,
        "restrictions": [],
        ...
    },
    {
        "pointer": "#/properties/group/properties/f1",
        "children": [],
        "type": "string",
        "restrictions": [],
        ...
    },
    {
        "pointer": "#/properties/group/properties/f2",
        "children": [],
        "type": "string",
        "restrictions": [],
        ...
    }
]
```

This represents an object, not an array. Once the user toggles "flere valg", setting `isArray` to true, we would expect the data to be updated to:

```json
[
    {
        "pointer": "#/properties/group",
        "children": [
            "#/properties/group/items/properties/f1",
            "#/properties/group/items/properties/f2"
        ],
        "type": "object",
        "isArray": true,
        "restrictions": [],
        ...
    },
    {
        "pointer": "#/properties/group/items/properties/f1",
        "children": [],
        "type": "string",
        "restrictions": [],
        ...
    },
    {
        "pointer": "#/properties/group/items/properties/f2",
        "children": [],
        "type": "string",
        "restrictions": [],
        ...
    }
]
```

We would also expect that the data returns to the original setup (without `items` in the path) if the `isArray` option is toggled _off_ again.

So, if user toggles `isArray`  _ON_, we want to update the child pointers where we replace the parent pointer with  `<parentPointer>/items`, f.eks. by doing 

```js
<childNodePointer>.replace('<parentNodePointer>', makePointer(<parentNodePointer>, Keyword.Items)
```